src/main/docs/guide/views/fieldset/fieldsetExample.adoc

```diff
@@ -0,0 +1,43 @@
+Imagine, you want to create an application which displays a form such as:
+
+[source, html]
+----
+<form action="/books/save" method="post">
+    <div class="mb-3">
+        <label for="title" class="form-label">Title</label>
+        <input type="text" name="title" value="" id="title" class="form-control" required="required"/>
+    </div>
+    <div class="mb-3">
+        <label for="pages" class="form-label">Pages</label>
+        <input type="number" name="pages" value="" id="pages" class="form-control" required="required"/>
```

Comment:
Why is there no `min` in this tag?  The BookSave field is annotated with `@Min(1)` and the inputnumber.html template has `th:min="${el.min()}"` but the generated source has no `min` attribute 🤔

Reply:
because, we don't generate those yet.

Reply:
Ahhh 😁